src/cloudant/account.py

```diff
@@ -392,32 +392,34 @@ def _usage_endpoint(self, endpoint, year=None, month=None):
         optional year and month URL elements.
 
         :param str endpoint: Cloudant usage endpoint.
-        :param int year: Year to query against.  Defaults to None.
-        :param int month: Month to query against.  Defaults to None.
-        """
-        if year is not None:
-            endpoint = posixpath.join(endpoint, str(year))
-        if month is not None:
-            if year is None:
-                raise CloudantException(
-                    (
-                        "must supply both year and month "
-                        "to usage endpoint: {0}"
-                    ).format(endpoint)
-                )
-            endpoint = posixpath.join(endpoint, str(month))
+        :param int year: Year to query against.  Optional parameter.
+        Defaults to None.  If used, it must be accompanied by the ``month``.
+        :param int month: Month to query against. Optional parameter.
+        Defaults to None.  If used, it must be accompanied by the ``year``.
+        """
+        if (year is not None) and (month is not None):
+            endpoint = posixpath.join(endpoint, str(year), str(month))
```

Comment:
If you're set on having this changed then I think it should be done as part of a separate PR since this is how we compose URLs throughout the library, so the change should probably be applied to the entire repo.  But fwiw, it doesn't seem like an odd use to me since the docs back it up and this way of constructing URLs was inherited from the original library written by the SAPI folks.  But it's your call @mikerhodes.